src/base/_defaults.scss

```diff
@@ -151,3 +151,26 @@ table {
   text-align: left;
   width: 100%;
 }
+
+/*
+ * 1. Removes default margin
+ * 2. Center content
+ */
+
+figure {
+  margin: 0; /* 1 */
+
+  & > * {
+    margin-left: auto; /* 2 */
+    margin-right: auto; /* 2 */
+  }
```

Comment:
Could you replace this with `text-align: center` on `figure`, which would also cascade to `figcaption`?